### PR TITLE
improve speed of site

### PR DESCRIPTION
**Pull request by StephenFluin**
remove jquery
minify html/css/js as part of build
compress images
defer JS

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -12,7 +13,24 @@ module.exports = function (eleventyConfig) {
...
   eleventyConfig.addPassthroughCopy({ _src: '/' });
 
+  // Allow us to put rendered markdown in HTML
   eleventyConfig.addPairedShortcode('markdown', (content) => {
     return rdmd.html(content);
   });
+
+  // Minify our build to save the bits
+  eleventyConfig.addTransform('htmlmin', function (content, outputPath) {
+    if (outputPath?.endsWith('.html')) {
```
Comment (alexroan) [inline .eleventy.js:23]: This line throwing an error locally:

```shell
Eleventy fatal error: (more in DEBUG output)
> Error in your Eleventy config file '/Users/alexroan/Documents/Projects/UIs/documentation/.eleventy.js'.

`EleventyConfigError` was thrown
> Unexpected token '.'

`SyntaxError` was thrown:
    /Users/alexroan/Documents/Projects/UIs/documentation/.eleventy.js:23
        if (outputPath?.endsWith('.html')) {
                       ^
    
    SyntaxError: Unexpected token '.'
        at wrapSafe (internal/modules/cjs/loader.js:915:16)
        at Module._compile (internal/modules/cjs/loader.js:963:27)
        at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
        at Module.load (internal/modules/cjs/loader.js:863:32)
        at Function.Module._load (internal/modules/cjs/loader.js:708:14)
        at Module.require (internal/modules/cjs/loader.js:887:19)
        at require (internal/modules/cjs/helpers.js:74:18)
        at TemplateConfig.mergeConfig (/Users/alexroan/Documents/Projects/UIs/documentation/node_modules/@11ty/eleventy/src/TemplateConfig.js:87:23)
        at new TemplateConfig (/Users/alexroan/Documents/Projects/UIs/documentation/node_modules/@11ty/eleventy/src/TemplateConfig.js:25:24)
        at Object.<anonymous> (/Users/alexroan/Documents/Projects/UIs/documentation/node_modules/@11ty/eleventy/src/Config.js:5:14)
error Command failed with exit code 1.
```

**Review by alexroan — CHANGES_REQUESTED** [review]

File: .eleventy.js (hunk 1 above)
Comment (StephenFluin) [reply .eleventy.js:23]: Looks like we need to specify a version of node so we are all on the same version. What version are you on?

Comment (StephenFluin) [reply .eleventy.js:23]: This is a newer is feature called optional chaining. much cleaner than the older

If a and a.b and a.b.c and a.b.c.d=...

Comment (StephenFluin) [reply .eleventy.js:23]: I double checked, you need at least node 14 (which is current https://nodejs.org/en/about/releases/). I've added this as a hard requirement in #67

**Comment by StephenFluin** [comment]
@alexroan rebased so it can be merged cleanly

**Review by alexroan — CHANGES_REQUESTED** [review]
The JS on ens.html doesn't work because ens.js still references jQuery.

**Comment by StephenFluin** [comment]
Great catch @alexroan , take another look. :)

**Review by alexroan — APPROVED** [review]

**Pull request merged**